adcc/ExcitedStates.py

```diff
@@ -38,6 +38,19 @@
 from .FormatDominantElements import FormatDominantElements
 
 
+class EnergyCorrection:
+    def __init__(self, name, function, description=None):
+        assert isinstance(name, str)
+        assert callable(function)
```

Comment:
Obviously should be replaced by better error handling :smile:.

Reply:
Done.